cli/collection.go

```diff
@@ -60,6 +62,29 @@ func MakeCollectionCommand(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if name != "" {
+				// Multiple params may have been specified, and in some cases both are needed.
+				// For example if a schema version and a collection name have been provided,
+				// we need to ensure tha a collection at the requested version is returned.
+				// Likewise we need to ensure that if a name and schema id are provided, but
+				// there are none matching both, that nothing is returned.
```

Comment:
thought: I don't think this comment reflect what happens in the switch statement. Only one of the three will ever be used.

Reply:
Yes, but that is due to how they interact.

If name and another param is provided, the other param will be used and name applied after.
If schema version is provided, schema does not matter.  We could check that the schema version fetched is actually of the given schema though in case someone provides impossible params.

Reply:
`// we need to ensure **that** a collection`

Reply:
> Ah yes ok I see what you mean

Am adding an error for that schema-schema version id clash

- [ ] add error

> we need to ensure **that** a collection

Cheers, will change :)

- [ ] collection name